app/models/item_type.rb

```diff
@@ -1,4 +1,5 @@
 class ItemType < ActiveRecord::Base
+  belongs_to :creator, class_name: Service
```

Comment:
It'd be good to have a validation for the presence of creator. Then you can remove that guard on the hook method.